assessment_store/openapi/api.yml

```diff
@@ -428,26 +428,6 @@ paths:
         schema:
           type: boolean
         required: false
-      - name: score
```

Comment:
This seemed unused - the open api specs will drift from reality now as they're not generated based on the flask app. We should remove them when we come to cleanup